frontend/src/app/components/tableWithFilter/table.component.html

```diff
@@ -0,0 +1,79 @@
+<p-table #datatable [value]="data" dataKey="id" [rows]="9" [showCurrentPageReport]="true"
+         [rowsPerPageOptions]="[10,25,50]" styleClass="p-datatable-customers p-datatable-gridlines"
+         [paginator]="true"
+         currentPageReportTemplate="Showing {first} to {last} of {totalRecords} entries"
+         [globalFilterFields]="['provinces','activeCases','deaths','hospitalizations']">
+  <ng-template pTemplate="caption">
+    <div class="p-d-flex">
+                <span class="p-input-icon-left p-ml-auto">
+                    <i class="pi pi-search"></i>
+                    <input pInputText type="text" (input)="datatable.filterGlobal($event.target, 'contains')" placeholder="Search keyword" />
+                </span>
+    </div>
+  </ng-template>
+  <ng-template pTemplate="header">
+    <tr>
+      <th>
+        <div class="p-d-flex p-jc-between p-ai-center">
+          Date
+          <p-columnFilter field="province" matchMode="in" display="menu" [showMatchModes]="false" [showOperator]="false" [showAddButton]="false">
+            <ng-template pTemplate="header">
+              <div class="p-px-3 p-pt-3 p-pb-0">
+                <span class="p-text-bold">Provinces Picker</span>
+              </div>
+            </ng-template>
+            <ng-template pTemplate="filter" let-value let-filter="filterCallback">
+              <p-multiSelect [ngModel]="value" [options]="provinces" placeholder="Any" (onChange)="filter($event)" optionLabel="provinces">
+                <ng-template let-option pTemplate="item">
+                  <div class="p-multiselect-province-option">
+                    <span class="p-ml-1">{{option}}</span>
+                  </div>
+                </ng-template>
+              </p-multiSelect>
+            </ng-template>
+          </p-columnFilter>
+        </div>
+      </th>
+
+      <th>
+        <div class="p-d-flex p-jc-between p-ai-center">
+          Active Cases Today
+          <p-columnFilter type="text" field="activeCases" display="menu"></p-columnFilter>
+        </div>
+      </th>
+      <th>
+        <div class="p-d-flex p-jc-between p-ai-center">
+          Deaths
+          <p-columnFilter field="deaths" type="text" display="menu"></p-columnFilter>
+        </div>
+      </th>
+      <th>
+        <div class="p-d-flex p-jc-between p-ai-center">
+          Hospitalizations
+          <p-columnFilter field="hospitalizations" type="text" display="menu"></p-columnFilter>
+        </div>
+      </th>
+    </tr>
+
+  </ng-template>
+
+
+  <ng-template pTemplate="body" let-data>
+    <tr>
+      <td>{{data.date}}</td>
+      <td>{{data.cases}}</td>
```

Comment:
Here you callthe property "cases", in the filtering header and the global filter you call it "activeCases". Some naming is wrong.

Reply:
Maybe also my fault, sorry

Reply:
active steht lei drinnen, damit des mal durchläuft, funkt ja aktuell eh nit er filter